### ISSUE TITLE
I can't uninstall nor update SpaceMonkey

### ISSUE THREAD

**Issue by Leynad777** [PHARTGAMES/SpaceMonkey#40]
Everytime I try this error-message appear and I also can't change permissions to this registry-thing...

![SpaceMonkey-Error](https://github.com/PHARTGAMES/SpaceMonkey/assets/150488233/fc32451b-1d43-4f6d-8321-31de0b49c25c)

**Comment by PHARTGAMES** [comment]
Yeah, I don't know why this happens.

There's a troubleshooter here that may fix the issue for this.

 https://support.microsoft.com/en-us/topic/fix-problems-that-block-programs-from-being-installed-or-removed-cca7d1b6-65a9-3d98-426b-e9f927e1eb4d

**Comment by Alkman222** [comment]
Same here...

**Comment by Leynad777** [comment]
> Same here...

Don't fix the permission!!! It's actually possible, but can't install Microsoft apps and games anymore and also not launch the ones I have. 

The fix is to uninstall the BeamNG plugin before in my case.

**Comment by PHARTGAMES** [comment]
@Leynad777 Does that troubleshooter mess up ms store?

**Comment by CephJedi** [comment]
Hi!   I just got a DOFReality H3 with a big interest in Squadrons, but I too cannot get SM to install on a windows 10 machine.   Same issue- the MS troublshooter can uninstall it, but I can't figure out how to get the installation to succeed.   Has there been any new discoveries with this issue?

**Comment by PHARTGAMES** [comment]
Hi, I've removed the installer completely now, different install steps, please try latest

https://github.com/PHARTGAMES/SpaceMonkey/releases/tag/v1.2.1

**Comment by M0nk3yofdoom** [comment]
> Hi, I've removed the installer completely now, different install steps, please try latest
> 
> https://github.com/PHARTGAMES/SpaceMonkey/releases/tag/v1.2.1

Should we uninstall previous version first?

**Comment by PerryHart** [comment]
Yeah uninstall the old first if you can, doesn't make much difference tho. The main thing is that you run the register.bat for the latest version so that the registry is updated to point to the location of the new version.
